src/translate.cpp

```diff
@@ -185,6 +179,10 @@ void Translate::preparePipelines(std::vector<std::unique_ptr<PipelineManager>>&
     }
     else
     {
+        if (outputFile.find("copc") != std::string::npos)
```

Comment:
use `ends_with(".copc")` to make sure we're not matching some string inside the path